packages/jsii-reflect/lib/type-system.ts

```diff
@@ -316,6 +316,18 @@ export class TypeSystem {
     return new Assembly(this, ass);
   }
 
+  private getAssemblyFile(pathToAssembly: string) {
+    const jsiiFile = path.join(pathToAssembly, '.jsii');
```

Comment:
There is a constant for `.jsii` called `SPEC_FILE_NAME` in `@jsii/spec`. It should be used.